modules/client-handler/src/main/java/org/apache/ignite/client/handler/ClientInboundMessageHandler.java

```diff
@@ -143,12 +152,17 @@ public class ClientInboundMessageHandler extends ChannelInboundHandlerAdapter {
     /** Context. */
     private ClientContext clientContext;
 
+    private ChannelHandlerContext channelHandlerContext;
```

Comment:
All fields have javadocs in this file. `authenticationManager` added below also has it. Let's be consistent and add javadoc here too.